__tests__/app.test.js

```diff
@@ -12,34 +13,119 @@ return seed(testData)
+            expect.objectContaining({ slug: 'euro game', description: 'Abstact games that involve little luck' }),
+            expect(categories.length).toBe(4)
 
         })
     });
 
   });
-  describe('Get/api/reviews', () => {
+  describe('Get /api/reviews', () => {
     test('Status 200 - Should respond with an array of review objects ', () => {
         return request(app)
         .get("/api/reviews")
         .expect(200)
-        .then(({})=>{
-
+        .then(({body})=>{
+            const { reviews } = body;
+            expect(reviews.length).toBe(13)
+            expect(reviews[0]).toBeInstanceOf(Object);
+            reviews.forEach((review)=>{
+                expect(review).toEqual(
+                    expect.objectContaining({
+                        review_id: expect.any(Number),
+                        title: expect.any(String),
+                        category: expect.any(String),
+                        designer: expect.any(String),
+                        owner: expect.any(String),
+                        review_img_url: expect.any(String),
+                        created_at: expect.any(String),
+                        votes: expect.any(Number),
+                        comment_count: expect.any(String)
+                    })
+                )
+            })
+        })
+        
+    });
+    test('should sort date by date in descending order', () => {
+        return request(app)
+        .get("/api/reviews")
+        .then(({body})=>{
+            const { reviews } = body;
+expect(reviews).toBeSortedBy('created_at',  {
+    descending: true,
+  })
 
         })
+    });
+  });
+  describe('Get /api/reviews/:review_id', () => {
+
+    test('should return a review object when given an id', () => {
         
+    
+    return request(app)
+    .get("/api/reviews/2")
+    .expect(200)
+    .then(({body})=>{
+        const { review } = body;
+        expect(review).toEqual(
+            expect.objectContaining({
+                review_id: 2,
+                title: 'Jenga',
+                category: 'dexterity',
+                designer: 'Leslie Scott',
+                owner: 'philippaclaire9',
+                review_body: 'Fiddly fun for all the family',
+                review_img_url:  'https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png',
+                created_at: expect.any(String),
+                votes: 5
+            })
+        )
+    })
+});
+test('should return a 404 error when given an id that doesnt exist', () => {
+    
+    return request(app)
+    .get("/api/reviews/4326")
+    .expect(404)
+    .then((({body:{msg}})=>{
+expect(msg).toBe('No review with that id')
+
+    }))
+    })
+    test('should return 400 when given an invalidId', () => {
+        return request(app)
+        .get("/api/reviews/chocolate")
+        .expect(400)
+        .then((({body:{msg}})=>{
+            expect(msg).toBe('Invalid input')
```

Comment:
"Invalid input" is not a very clear error message for a user to receive - its implies that this is  a POST or PATCH, which it isn't. 
"bad request" although very generic - might be clearer to help someone understand what has gone wrong.